### ISSUE TITLE
logo contribution

### ISSUE THREAD

**Issue by reallinfo** [igas/faker#250]
Hi, I contribute to open source softwares by designing logos. I designed a logo for faker. The idea of ​​logo is available in the following image.  I will wait for feedback. Thank you.

![faker](https://user-images.githubusercontent.com/36298335/55498974-ddd8b600-566a-11e9-854c-0f7f10673f4c.jpg)

**Comment by vbrazo** [comment]
Loved the idea of adding a logo to the repository 👍

**Comment by anthonator** [comment]
They look great! This is probably a decision for @igas though.

**Comment by igas** [comment]
hey, I love the idea. And appreciate the work. I have not dealt with design in OSS, usually, we go through a few iterations. I see the logic in the idea, but I don't associate it with project personally. I like simplicity but would go in the direction of the abstract meaning of the word itself. For me, faker is more like a pretender. I would play around ball masks with/without holder or theatrical masks. Maybe mime. Does anyone else have any associations?

**Comment by reallinfo** [comment]
Hi again, I have designed a logo by going over your ideas. I did some research on the internet. only if I act with the idea of ​​theater mask, the logo would be very general. That's why I included the functionality of the software in the logo idea. and I combined the theater icon with a data icon. What do you think?
![FAKER](https://user-images.githubusercontent.com/36298335/56805184-e323bd80-6830-11e9-9cff-4401a0b36933.jpg)

**Comment by vbrazo** [comment]
I'd prefer to see smart eyes or perhaps evil ones. Not sure if happiness could express the library's core idea.

**Comment by vbrazo** [comment]
We'll also need a logo for the community and are open for suggestions.

**Comment by vbrazo** [comment]
@igas what about adding a raw elixir logo in the GitHub community like the one below? 

![elixir](https://user-images.githubusercontent.com/1292556/67709905-2b589f80-f97c-11e9-82ed-c371a9ca2408.png)

**Comment by vbrazo** [comment]
I would also add a description for the community. Make it a real community with a clear goal 🚀

**Comment by igas** [comment]
I don't think we can use official logo due to licence. What do you think about this one? 

https://pixabay.com/photos/bottles-potions-ink-dye-magic-4387824/

> Free for commercial use
> No attribution required

Feel free to write up one, I'm happy to add it.

**Comment by vbrazo** [comment]
looks good to me 👍

**Comment by vbrazo** [comment]
I like the logo @igas we should probably add a description too. Something like: `Group for shared work on Elixir libraries and documentation`.
